tests/store/artifact/test_mlflow_artifact_repo.py

```diff
@@ -22,36 +22,43 @@ def test_artifact_uri_factory():
     assert isinstance(repo, MlflowArtifactsRepository)
 
 
-def test_mlflow_artifact_uri_formats_resolved():
-    base_url = "/api/2.0/mlflow-artifacts/artifacts"
-    base_path = "/my/artifact/path"
-    conditions = [
-        (
-            f"mlflow-artifacts://myhostname:4242{base_path}/hostport",
-            f"http://myhostname:4242{base_url}{base_path}/hostport",
-        ),
-        (
-            f"mlflow-artifacts://myhostname{base_path}/host",
-            f"http://myhostname{base_url}{base_path}/host",
-        ),
-        (
-            f"mlflow-artifacts:{base_path}/nohost",
-            f"http://localhost:5000{base_url}{base_path}/nohost",
-        ),
-        (
-            f"mlflow-artifacts://{base_path}/redundant",
-            f"http://localhost:5000{base_url}{base_path}/redundant",
-        ),
-        (
-            "mlflow-artifacts:/",
-            f"http://localhost:5000{base_url}",
-        ),
-    ]
+base_url = "/api/2.0/mlflow-artifacts/artifacts"
+base_path = "/my/artifact/path"
+conditions = [
+    (
+        f"mlflow-artifacts://myhostname:4242{base_path}/hostport",
+        f"http://myhostname:4242{base_url}{base_path}/hostport",
+        "http://myhostname:4242",
```

Comment:
```suggestion
        f"mlflow-artifacts://myhostname:4242{base_path}/hostport",
        "http://myhostname:4242",
        f"http://myhostname:4242{base_url}{base_path}/hostport",
```

Can we fix the order?

Reply:
I like your edited way in the last comment better. Changed to that easier to read and cleaner implementation.